tests/magnetostatics/setup_methods.py

```diff
@@ -28,7 +28,7 @@ def _plot_verification_test(
 ):
     def relative_diff(x1, x2):
         diff = np.zeros(x1.shape)
-        mask = np.where(x1 != 0)
+        mask = np.nonzero(x1 != 0)
```

Comment:
There are a number of instances where np.nonzero(x != 0) is called.

Surely it would just be np.nonzero(x)?